File: src/Entity/Livreur.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Livreur
 *
 * @ORM\Table(name="livreur")
 * @ORM\Entity(repositoryClass="App\Repository\LivreurRepository")
 */
class Livreur
{
    /**
     * @var int
     *
     * @ORM\Column(name="idLivreur", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idlivreur;

    /**
     * @var string
     *
     * @ORM\Column(name="nomLivreur", type="string", length=225, nullable=false)
     */
    private $nomlivreur;

    /**
     * @var string
     *
     * @ORM\Column(name="prenomLivreur", type="string", length=225, nullable=false)
     */
    private $prenomlivreur;

    /**
     * @var string
     *
     * @ORM\Column(name="email", type="string", length=225, nullable=false)
     */
    private $email;

    /**
     * @var int
     *
     * @ORM\Column(name="numTelephoneLivreur", type="integer", nullable=false)
     */
    private $numtelephonelivreur;

    /**
     * @var string
     *
     * @ORM\Column(name="adresseLivreur", type="string", length=255, nullable=false)
     */
    private $adresselivreur;

    public function getIdlivreur(): ?int
    {
        return $this->idlivreur;
    }

    public function getNomlivreur(): ?string
    {
        return $this->nomlivreur;
    }

    public function setNomlivreur(string $nomlivreur): self
    {
        $this->nomlivreur = $nomlivreur;

        return $this;
    }

    public function getPrenomlivreur(): ?string
    {
        return $this->prenomlivreur;
    }

    public function setPrenomlivreur(string $prenomlivreur): self
    {
        $this->prenomlivreur = $prenomlivreur;

        return $this;
    }

    public function getEmail(): ?string
    {
        return $this->email;
    }

    public function setEmail(string $email): self
    {
        $this->email = $email;

        return $this;
    }

    public function getNumtelephonelivreur(): ?int
    {
        return $this->numtelephonelivreur;
    }

    public function setNumtelephonelivreur(int $numtelephonelivreur): self
    {
        $this->numtelephonelivreur = $numtelephonelivreur;

        return $this;
    }

    public function getAdresselivreur(): ?string
    {
        return $this->adresselivreur;
    }

    public function setAdresselivreur(string $adresselivreur): self
    {
        $this->adresselivreur = $adresselivreur;

        return $this;
    }


}

File: src/com/mycompany/myap/entities/Livreur.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.myap.entities;

/**
 *
 * @author azizm
 */
public class Livreur {
    private int idLivreur ;
    private String nomLivreur ; 
    private String prenomLivreur ; 
    private String email ;
    private String adresseLivreur ;
    private int numTelephoneLivreur ;

    public int getIdLivreur() {
        return idLivreur;
    }

    public void setIdLivreur(int idLivreur) {
        this.idLivreur = idLivreur;
    }

    public String getNomLivreur() {
        return nomLivreur;
    }

    public void setNomLivreur(String nomLivreur) {
        this.nomLivreur = nomLivreur;
    }

    public String getPrenomLivreur() {
        return prenomLivreur;
    }

    public void setPrenomLivreur(String prenomLivreur) {
        this.prenomLivreur = prenomLivreur;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getAdresseLivreur() {
        return adresseLivreur;
    }

    public void setAdresseLivreur(String adresseLivreur) {
        this.adresseLivreur = adresseLivreur;
    }

    public int getNumTelephoneLivreur() {
        return numTelephoneLivreur;
    }

    public void setNumTelephoneLivreur(int numTelephoneLivreur) {
        this.numTelephoneLivreur = numTelephoneLivreur;
    }

    public Livreur(String nomLivreur, String prenomLivreur, String email,  int numTelephoneLivreur,String adresseLivreur) {
        this.nomLivreur = nomLivreur;
        this.prenomLivreur = prenomLivreur;
        this.email = email;
         this.numTelephoneLivreur = numTelephoneLivreur;
        this.adresseLivreur = adresseLivreur;
    }

    public Livreur(int idLivreur, String nomLivreur, String prenomLivreur, String email,int numTelephoneLivreur, String adresseLivreur ) {
        this.idLivreur = idLivreur;
        this.nomLivreur = nomLivreur;
        this.prenomLivreur = prenomLivreur;
        this.email = email;  
        this.numTelephoneLivreur = numTelephoneLivreur;
        this.adresseLivreur = adresseLivreur;
    }

    public Livreur(String nomLivreur, String prenomLivreur, String email, String adresseLivreur, int numTelephoneLivreur) {
        this.nomLivreur = nomLivreur;
        this.prenomLivreur = prenomLivreur;
        this.email = email;
        this.adresseLivreur = adresseLivreur;
        this.numTelephoneLivreur = numTelephoneLivreur;
    }

    
    public Livreur() {
    }

    @Override
    public String toString() {
        return "Livreur{" + "idLivreur=" + idLivreur + ", nomLivreur=" + nomLivreur + ", prenomLivreur=" + prenomLivreur + ", email=" + email + ", adresseLivreur=" + adresseLivreur + ", numTelephoneLivreur=" + numTelephoneLivreur + '}';
    }

    
    
    
}